diff --git a/go.mod b/go.mod
index 239b4cec..9cbf4016 100644
--- a/go.mod
+++ b/go.mod
@@ -83,3 +83,4 @@ require (
 )
 
 replace github.com/DataDog/dd-trace-go v0.6.1 => github.com/datadog/dd-trace-go v0.6.1
+replace github.com/metalkube/coredns-mdns => /go/src/github.com/metalkube/coredns-mdns/
diff --git a/plugin.cfg b/plugin.cfg
index a9450d50..f8d67b50 100644
--- a/plugin.cfg
+++ b/plugin.cfg
@@ -37,6 +37,7 @@ dnstap:dnstap
 chaos:chaos
 loadbalance:loadbalance
 cache:cache
+mdns:github.com/metalkube/coredns-mdns
 rewrite:rewrite
 dnssec:dnssec
 autopath:autopath
